api/installations_service.go

```diff
@@ -0,0 +1,138 @@
+package api
+
+import (
+	"encoding/json"
+	"fmt"
+	"net/http"
+	"net/http/httputil"
+	"strings"
+)
+
+const (
+	StatusRunning   = "running"
+	StatusSucceeded = "succeeded"
+	StatusFailed    = "failed"
+)
+
+type InstallationsService struct {
+	client httpClient
+}
+
+type InstallationsServiceOutput struct {
+	ID     int
+	Status string
+	Logs   string
+}
+
+func NewInstallationsService(client httpClient) InstallationsService {
+	return InstallationsService{
+		client: client,
+	}
+}
+
```

Comment:
While I understand the motivation for the installations service to only have stateless functions like Trigger, Logs, Status, and have the actual command deal with tracking the status, iterating through logs, etc...I'm just wondering if a lot of that stateful business logic should instead be part of the service? This way the command can be lightweight, and would only be responsible for passing the parameters down to the service itself. I feel like this respects objects boundaries better, but you may have a different opinion on this.